ERP/resources/views/components/navbar.blade.php

```diff
@@ -13,6 +13,14 @@
                 <li class="nav-item">
                     <a class="nav-link active" href="{{route('user.management')}}">User Management</a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('inventory')}}">Inventory</a>
+                </li>
+                @endif
+                @if(Auth::user()->user_type==4)
+                    <li class="nav-item">
+                        <a class="nav-link active" href="{{route('inventory')}}">Inventory</a>
+                    </li>
```

Comment:
This makes sense, the inventory navbar entry shows up only for the inventory person